vuepress/docs/next/docs/releases/README.md

```diff
@@ -133,6 +133,9 @@ Entando 7.1.0 is an important feature release that introduces major changes to t
 - Fixed CVE-2022-33879 (apache tika, DoS)
 - Fixed CVE-2022-24823 (apache netty, Information Exposure)
 
+## Deprecation Warnings
+- The runtime option to disable the AppBuilder (`APPBUILDERINTEGRATIONENABLED: "false"`) and use the AdminConsole to manage an Entando Application is now deprecated and may be removed in a future release.
```

Comment:
i feel like removing spaces in "App Builder" and "Admin Console" is intentional, but please refresh me on the reasoning and difference between spaced vs unspaced?